### PR TITLE
fix: remove groups from footer in english

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

ref: https://github.com/freeCodeCamp/freeCodeCamp/issues/52259

### CODE REVIEW

File: config/i18n/locales/french/translations.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
   "embed-title": "Embedded content",
   "footer": {
     "tax-exempt-status": "freeCodeCamp is a donor-supported tax-exempt 501(c)(3) charity organization (United States Federal Tax Identification Number: 82-0779546)",
-    "mission-statement": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.",
+    "mission-statement": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public.",
```
Comment (sidemt) [inline config/i18n/locales/french/translations.json]: The files other than English will be updated through Crowdin. So we don't need to update it here.

File: config/i18n/locales/korean/translations.json [hunk 2]
```diff
@@ -69,7 +69,7 @@
   "embed-title": "Embedded content",
   "footer": {
     "tax-exempt-status": "freeCodeCamp is a donor-supported tax-exempt 501(c)(3) charity organization (United States Federal Tax Identification Number: 82-0779546)",
-    "mission-statement": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.",
+    "mission-statement": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public.",
```
Comment (sidemt) [inline config/i18n/locales/korean/translations.json]: The files other than English will be updated through Crowdin. So we don't need to update it here.

**Review by sidemt — CHANGES_REQUESTED** [review]

File: config/i18n/locales/french/translations.json (hunk 1 above)
Comment (ahmaxed) [inline config/i18n/locales/french/translations.json]:
```suggestion
    "mission-statement": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.",
```

File: config/i18n/locales/korean/translations.json (hunk 2 above)
Comment (ahmaxed) [inline config/i18n/locales/korean/translations.json]:
```suggestion
    "mission-statement": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.",
```

**Review by sidemt — APPROVED** [review]
LGTM

**Pull request merged**